data/graphics.js

```diff
@@ -10009,11 +10058,60 @@ var BattleMoveAnims = {
 			}, 'decel', 'explode');
 		}
 	},
+	hydroshot: {
```

Comment:
Otherwise this should be "named" as one of the moves that use this animation, which I think is the preferred option right now. Unless you also update scald and hydro pump to call this hydro shot animation